designer/server/src/test/scala/pl/touk/nussknacker/ui/api/NodesApiSpec.scala

```diff
@@ -0,0 +1,1093 @@
+                 |        "parameters": [
+                 |            {
+                 |                "name": "id",
+                 |                "expression": {
+                 |                    "language": "spel",
+                 |                    "expression": "'a'"
+                 |                }
+                 |            }
+                 |        ]
+                 |    },
+                 |    "output": "out",
+                 |    "additionalFields": null,
+                 |    "type": "Enricher"
+                 |}""".stripMargin
+            )
+            .when()
+            .post(s"$nuDesignerHttpAddress/api/nodes/${process.name}/additionalInfo")
+            .Then()
+            .statusCode(200)
+            .body(
+              equalsJson(s"""{
+                   |  "content": "\\nSamples:\\n\\n| id  | value |\\n| --- | ----- |\\n| a   | generated |\\n| b   | not existent |\\n\\nResults for a can be found [here](http://touk.pl?id=a)\\n",
+                   |  "type": "MarkdownAdditionalInfo"
+                   |}""".stripMargin)
+            )
+
+          given()
+            .basicAuth("allpermuser", "allpermuser")
+            .and()
+            .jsonBody(
+              s"""{
+                 |    "id": "1",
+                 |    "service": {
+                 |        "id": "otherService",
+                 |        "parameters": []
+                 |    },
+                 |    "output": "out",
+                 |    "additionalFields": null,
+                 |    "type": "Enricher"
+                 |}""".stripMargin
+            )
+            .when()
+            .post(s"$nuDesignerHttpAddress/api/nodes/${process.name}/additionalInfo")
+            .Then()
+            .statusCode(200)
+            .body(
+              equalTo("")
+            )
+        }
+        "return 404 for not existent process" in {
+          val wrongName: String = "wrongProcessName"
+
+          given()
+            .basicAuth("allpermuser", "allpermuser")
+            .and()
+            .jsonBody(
+              s"""{
+                 |    "id": "1",
+                 |    "service": {
+                 |        "id": "otherService",
+                 |        "parameters": []
+                 |    },
+                 |    "output": "out",
+                 |    "additionalFields": null,
+                 |    "type": "Enricher"
+                 |}""".stripMargin
+            )
+            .when()
+            .post(s"$nuDesignerHttpAddress/api/nodes/$wrongName/additionalInfo")
+            .Then()
+            .statusCode(404)
+            .body(
+              equalTo(s"No scenario $wrongName found")
+            )
+        }
+      }
+      "not authenticated should" - {
+        "forbid access" in {
+          createSavedProcess(process, TestCategories.Category1, TestProcessingTypes.Streaming)
+
+          given()
+            .auth()
+            .none
```

Comment:
it would be nice to have `noAuth` extension too ;)

Reply:
and I started to wonder if in case of "forbid access" cases we should pass no credentials or maybe pass invalid credentials for some existing user? WDYT?

Reply:
I have been using `auth.none` as I thought if user wasn't authenticated, it wouldn't matter if it was due to wrong or no credentials and I thought that `auth.none` would be better visible in code than some wrong credentials, but I am open to change if we think it's better the other way